commands/initial/templates/package.json

```diff
@@ -2,11 +2,11 @@
   "name": "{{ name }}",
   "version": "{{ version }}",
   "description": "{{ readmeTitle }}, an Angular library",
-  "main": "dist/index.js",
-  "jsnext:main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "dist/{{ name }}.bundle.js",
+  "jsnext:main": "dist/{{ name }}.bundle.js",
+  "types": "dist/src/index.d.ts",
   "scripts": {
-    "build": "ngc",
+    "build": "webpack --config ./webpack/webpack.build.js",
```

Comment:
we need `ngc` to run to get the `*.metadata.json` files so that the produced libraries can be used with AoT...see: http://www.usefuldev.com/blog/post/using-ngc-to-build-an-angular-2-library-project